neurobooth_os/server_acq.py

```diff
@@ -153,6 +148,11 @@ def run_acq(logger):
                 lowFeed.close()
                 lowFeed_running = False
                 print("Closing RTD cam")
+
+            if system_resource_logger is not None:
+                system_resource_logger.stop()
+            logging.shutdown()
```

Comment:
Changing things so that logging is the last thing to go. I don't recall if there was a good reason for it being up top.